research/frontend/css/sign-up.css

```diff
@@ -0,0 +1,21 @@
+body {
```

Comment:
I see a lot of common styles between css files. Usually it's preferred to have a common.css that holds basic styles and let each css file overwrite it. But I'm keeping it this way :) Up to you!